auto.py

```diff
@@ -13,27 +14,46 @@
 class Autonomous(commands2.SequentialCommandGroup):
     def __init__(self, drive : Westcoast, navx : navx.AHRS, armController : ArmController, grabber : Grabber, position : int) -> None:
         super().__init__()
-       
+
         distance1 = wpilib.SmartDashboard.getNumber("Auto Distance 1", 7.25)
         distance2 = wpilib.SmartDashboard.getNumber("Auto Distance 2", 7.25)
-        turnAngle = 180
+        turnAngle = 30
 
         log.info(f"Auto Distance 1: {distance1}")
         log.info(f"Auto Distance 2: {distance2}")
         log.info(f"Auto Turn Angle: {turnAngle}")
-       
+
         if (position == EPosition.CENTER):
```

Comment:
When you get a chance, you should probably add comments to what these autos do ie "Center: Move the arm to mid cone placement, release code and drive to the charging station and balance" or whatever

Reply:
```suggestion
        #TODO https://github.com/Raptacon/Robot-2023/pull/152/files#r1148391001 
        if (position == EPosition.CENTER):
```